### PR TITLE
[14.0][mig] fetchmail_notify_error_to_sender : deprecated field remove

### PR DESCRIPTION
**Pull request by hkapatel-initos**
Migration to 14.0

### CODE REVIEW

**Comment by hkapatel-initos** [comment]
@AaronHForgeFlow can you please review this PR ?

**Comment by hkapatel-initos** [comment]
@MiquelRForgeFlow @AaronHForgeFlow could you, please review this pull request?

File: fetchmail_notify_error_to_sender/__manifest__.py
Comment (MiquelRForgeFlow) [inline fetchmail_notify_error_to_sender/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: fetchmail_notify_error_to_sender/models/mail_thread.py
Comment (MiquelRForgeFlow) [inline fetchmail_notify_error_to_sender/models/mail_thread.py]:
```suggestion
                .with_context(sender_message=message, route_exception=ve)
```
This way context is not being reset.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM, after fixing @MiquelRForgeFlow comments

**Review by atchuthan — APPROVED** [review]

File: fetchmail_notify_error_to_sender/models/mail_thread.py
Comment (hkapatel-initos) [reply fetchmail_notify_error_to_sender/models/mail_thread.py]: @MiquelRForgeFlow  and @AaronHForgeFlow I have done this can u please give me the approval label.

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by hkapatel-initos** [comment]
@archetipo can you please try to merge this pr.

**Review by dsolanki-initos — APPROVED** [review]
LGTM. Code reivew

**Comment by hkapatel-initos** [comment]
@pedrobaeza can you please review and merge the PR?

**Comment by pedrobaeza** [comment]
Merging blindly based on current approves:

/ocabot merge nobump

**Pull request merged**
